charts/gateway-addons-helm/values.yaml

```diff
@@ -1,5 +1,12 @@
+tags:
```

Comment:
curious why we need this ?

Reply:
to provide users more options to play with, also indicates which component is all about metrics or logging etc.

Reply:
ah TIL about this https://helm.sh/docs/topics/charts/#tags-and-condition-fields-in-dependencies :)